src/components/AutoSuggest/AutoSuggest.test.tsx

```diff
@@ -99,35 +115,33 @@ describe('src/components/AutoSuggest', () => {
       jest.advanceTimersByTime(INPUT_DELAY)
     })
 
-    await findByTestId('autoSuggest')
+    await screen.findByTestId('autoSuggest')
 
     expect(fetch).not.toHaveBeenCalled()
 
     act(() => {
       fireEvent.change(input, { target: { value: 'Ams' } })
     })
 
-    await findByTestId('autoSuggest')
+    await screen.findByTestId('autoSuggest')
 
     expect(fetch).not.toHaveBeenCalled()
 
```

Comment:
I don't understand why fetch should not be called here.

Reply:
Fetch should not be called when the input delay timer hasn't passed